tests/src/testAssembler.cpp

```diff
@@ -31,8 +39,9 @@ auto SimpleAssemblersTest() {
   std::array<int, 2> elementsPerDirection = {2, 1};
   auto grid                               = std::make_shared<Grid>(bbox, elementsPerDirection);
 
-  for (int i = 0; i < 4; ++i) {
-    auto gridView = grid->leafGridView();
+  for (int ref = 0; ref < 4; ++ref) {
+    auto gridView  = grid->leafGridView();
+    auto totalDOFs = 2 * gridView.size(2);
 
     using namespace Dune::Functions::BasisFactory;
     auto basis        = Ikarus::makeBasis(gridView, power<2>(lagrange<1>()));
```

Comment:
Can you also add check for quadratic basis. To make sure for edge dofs everything also works?
You can use the runtime version `lagrange(k)`. and wrap everything in a new function maybe. and the in these test your can write subtests.

Reply:
Then you can add two tests at https://github.com/ikarus-project/ikarus/blob/da5a5f61e57a78d52f81772d352fe584bc84f949/tests/src/testAssembler.cpp#L157C3-L157C37